src/EditorFeatures/CSharpTest2/Recommendations/EnableKeywordRecommenderTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Recommendations
+{
+    [Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+    public class EnableKeywordRecommenderTests : KeywordRecommenderTests
```

Comment:
💡 Would be good to verify it's not offered on the next line:

```
#nullable
  $$
```

Reply:
Investigating...